tests/lib/snaps/test-snapd-base-bare/meta/snap.yaml

```diff
@@ -0,0 +1,4 @@
+name: test-snapd-base-bare
+version: 1.0
+type: base
```

Comment:
this new type isn't added in any snapcraft PR, only the `base` field, but not the `base` snap `type`